app/src/main/res/layout/dialog_confirmation.xml

```diff
@@ -0,0 +1,78 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:tools="http://schemas.android.com/tools"
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto">
+    <LinearLayout
```

Comment:
@danacealfonso 
Can you convert it to ConstraintLayout?

Reply:
Change layouts in 566294d8722aefaa229b9c9c8f481a16fe5f72cf